core/src/environments/godot_environment.py

```diff
@@ -1,65 +1,111 @@
+import json
+
 import numpy as np
-from gymnasium.spaces import Box, Discrete
+from gymnasium.spaces import Box, Dict
+from ray.rllib.env.multi_agent_env import MultiAgentEnv
+from ray.rllib.utils.typing import MultiAgentDict
 
-from core.src.environments.environment import Environment
 from core.src.settings import get_settings
 from core.src.utils.godot_handler import GodotHandler
 
 environment_settings = get_settings().environment
 
 
-class GodotServerEnvironment(Environment[np.ndarray, np.integer]):
+class GodotServerEnvironment(MultiAgentEnv):
+    action_space = Dict(
+        {
+            "accelerate": Box(
+                low=environment_settings.observation_space_low,
+                high=environment_settings.observation_space_high,
+                shape=(),
+                dtype=np.float32,
+            ),
+            "rotate": Box(
+                low=environment_settings.observation_space_low,
+                high=environment_settings.observation_space_high,
+                shape=(),
+                dtype=np.float32,
+            ),
+        }
+    )
 
-    action_space = Discrete(get_settings().environment.action_space_range)
     observation_space = Box(
-        low=environment_settings.observation_space_low,
-        high=environment_settings.observation_space_high,
-        shape=(get_settings().environment.observation_space_size,),
-        dtype=np.float32
+        low=np.array([-(2**60)] * environment_settings.observation_space_size),
+        # needs to be changed to observation_space_low/high when it's fixed in godot
+        high=np.array([2**60] * environment_settings.observation_space_size),
```

Comment:
If all dimensions of our observation space have the same low and high bounds I think it's simpler to just pass in 
scalar values instead of arrays.
```suggestion
        low=-2**60,
        # needs to be changed to observation_space_low/high when it's fixed in godot
        high=2**60,
```